linux/setup_repo.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# This script installs a Jamulus repository to Debian based systems
+
+if [[ ${EUID} -ne 0 ]]; then
+     echo "Error: This script must be run as root."
+     exit 1
+fi
+
+REPO_FILE=/etc/apt/sources.list.d/jamulus.list
+KEY_FILE=/etc/apt/trusted.gpg.d/jamulus.asc
+GITHUB_REPOSITORY="jamulussoftware/jamulus"
+
+echo "Setting up Jamulus repo at ${REPO_FILE}..."
+echo "deb https://github.com/${GITHUB_REPOSITORY}/releases/latest/download/ ./" > ${REPO_FILE}
```

Comment:
This uses a trick: latest points to the last release tagged as "stable" by GitHub. Thus, the link remains unchanged, but a redirect guarantees the correct source.